templates/webhook.bin.erb

```diff
@@ -181,15 +182,23 @@ end
     end
 
     def run_command(command)
-      if Open3.respond_to?('capture3')
-        stdout, stderr, exit_status = Open3.capture3(command)
-        message = "triggered: #{command}\n#{stdout}\n#{stderr}"
-      else
-      message = "forked: #{command}"
-        Process.detach(fork{ exec "#{command} &"})
-        exit_status = 0
+      message = ''
+      File.open(LOCKFILE, 'w+') do |file|
+        # r10k has a small race condition which can cause failed deploys if two happen
+        # more or less simultaneously. To mitigate, we just lock on a file and wait for
+        # the other one to complete.
+        file.flock(File::LOCK_EX)
+
+        if Open3.respond_to?('capture3')
```

Comment:
Oh god, can we please, please let Ruby 1.8.7 die? Ruby 1.9.3 has this (https://ruby-doc.org/stdlib-1.9.3/libdoc/open3/rdoc/Open3.html#method-c-capture3), and even that version is EOL'd.

Please, Ruby 1.8.7 is old and tired. It just wants to go out to pasture, graze quietly, and enjoy the twilight years of its life. Stop making Ruby 1.8.7 suffer on.